rust/src/srtp_parameters.rs

```diff
@@ -35,3 +53,32 @@ impl Default for SrtpCryptoSuite {
         Self::AesCm128HmacSha180
     }
 }
+
+impl ToString for SrtpCryptoSuite {
+    fn to_string(&self) -> String {
+        match self {
+            Self::AeadAes256Gcm => String::from("AEAD_AES_256_GCM"),
+            Self::AeadAes128Gcm => String::from("AEAD_AES_128_GCM"),
+            Self::AesCm128HmacSha180 => String::from("AES_CM_128_HMAC_SHA1_80"),
+            Self::AesCm128HmacSha132 => String::from("AES_CM_128_HMAC_SHA1_32"),
+        }
+    }
+}
+
+/// Error that caused [`SrtpCryptoSuite`] parsing error.
+#[derive(Debug, Eq, PartialEq)]
+pub struct ParseCryptoSuiteError;
```

Comment:
This will be removed on the next PR by using SrtpCryptoSuite type in flatbuffers.